lib/lighthouse-reporter.js

```diff
@@ -132,6 +132,8 @@ const createBudgetsReport = (results) => {
       results.audits['performance-budget'].details.items) ||
     [];
 
+  const timings = results?.audits?.['timing-budget']?.details?.items || [];
```

Comment:
Should I refactor this part so that users dont need to have node14?

Reply:
maybe makes sense actually